magento/magento2
# [Admin][improvement] Do not allow HTML tags for product attribute options when type is dropdown or multiple select

### Description (*)
This PR aims to add a new validation for the options of the product attribute type "Dropdown", "Multiple Select" that checks whenever the option value contains HTML tags.

HTML tags are not useful nor displayed on the storefront for the product attribute options, those the store manager should be aware of it.

### Related Pull Requests
https://github.com/magento/magento2/pull/27371

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)

1. Go to the Admin area
2. Go to Stores -> Attributes -> Product
3. Press the button "Add New Attribute"
4. Fill in the attribute default label
5. Select the "Catalog Input Type for Store Owner" to "Dropdown" or "Multiple Select"
6. In the section "Manage Options (Values of Your Attribute)" below add new options that contain HTML tags like `<span>`, `</strong>`, etc.
7. Press the "Save and Continue Edit" or "Save Attribute" button
8. See the error message
![Error message that says the option contains HTML tags](https://user-images.githubusercontent.com/13456702/77455366-6d3e2280-6e02-11ea-94a8-dc5f533ede98.png)


### Questions or comments
Are the provided Unit and MFTF tests enough or there should be done more?

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29602: [Admin][improvement] Do not allow HTML tags for product attribute options when type is dropdown or multiple select
